test/ci/analysis.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+set -ve
+
+MACHINE=$(uname -m)
+uname -a
+gcc --version
+clang --version
```

Comment:
`${CC} --version`, no? The rest of the script is compiler agnostic.

Reply:
`clang` is used explicitly in `make analyze`
I changed `gcc` by `${CC}`